src/routes/index.ts

```diff
@@ -28,6 +28,10 @@ export class Routes {
       res.send({ msg: 'Server is up and running' });
     });
 
+    this.app.use('/emoji', async (req, res) => {
+      res.sendFile(path.resolve('./emoji/' + req.url));
```

Comment:
можно тут указать пусть через `__dirname`?
```suggestion
      res.sendFile(path.resolve(__dirname, '..', '..', 'emoji', req.url));
```

Reply:
на вопрос "зачем" отвечу, что так понятнее, куда лезем